### PR TITLE
Updated introduction section

### PR DESCRIPTION
**Pull request by andy-maier**

### CODE REVIEW

File: docs/source/intro.rst [hunk 1]
```diff
@@ -113,169 +131,107 @@ The following sections describe these two installation methods.
...
 
-5. Set the ANSIBLE_LIBRARY environment variable by sourcing the script created
-   in step 4:
+5.  Install the ibm.zhmc Ansible Galaxy collection:
 
-   .. code-block:: bash
+    .. code-block:: bash
 
-      $ . setup_ansible_library
+        $ ansible-galaxy colletion install ibm.zhmc
```
Comment (sabyadi) [inline docs/source/intro.rst:222]: A typo in the word "collection".

File: docs/source/intro.rst [hunk 2]
```diff
@@ -113,169 +131,107 @@ The following sections describe these two installation methods.
...
 
-   This must be done after each switch (or deactivation) of the active Python
-   environment and before any Ansible command (that uses these modules) is
-   invoked.
+    This will install the collection to ``$HOME/.ansible/collections`` and its
+    dependent Python packages into the active Python (i.e. the virtual Python
```
Comment (sabyadi) [inline docs/source/intro.rst:225]: Are you sure the installation will install the dependencies?

File: docs/source/intro.rst [hunk 3]
```diff
@@ -377,11 +334,11 @@ Then, run the playbooks:
 Versioning
 ----------
 
-This documentation applies to version |release| of the zhmc-ansible-modules
-package. You can also see that version in the top left corner of this page.
+This documentation applies to version |release| of the ibm.zhmc
```
Comment (sabyadi) [inline docs/source/intro.rst:337]: Is |release| some type of a placeholder?

File: docs/source/intro.rst [hunk 4]
```diff
@@ -396,17 +353,17 @@ Compatibility
...
-zhmc-ansible-modules package means that the user can safely upgrade to that new
+Ansible playbook using it. Thus, a backwards compatible new version of an
+Ansible Galaxy collection means that the user can safely upgrade to that new
 version without encountering compatibility issues in any Ansible playbooks
-using these modules.
+using the modules in the collection.
 
-This package uses the rules of `Semantic Versioning 2.0.0`_ for compatibility
+This collection uses the rules of `Semantic Versioning 2.0.0`_ for compatibility
 between package versions, and for :ref:`deprecations <Deprecations>`.
```
Comment (sabyadi) [inline docs/source/intro.rst:362]: Broken link.

File: docs/source/intro.rst [hunk 5]
```diff
@@ -377,11 +334,11 @@ Then, run the playbooks:
 Versioning
 ----------
 
-This documentation applies to version |release| of the zhmc-ansible-modules
-package. You can also see that version in the top left corner of this page.
+This documentation applies to version |release| of the ibm.zhmc
+Ansible Galaxy collection. You can also see that version in the top left corner
+of this page.
```
Comment (sabyadi) [inline docs/source/intro.rst:339]: The generated docs do not seem to have a version in the top left corner.

File: docs/source/intro.rst [hunk 6]
```diff
@@ -396,17 +353,17 @@ Compatibility
...
-Ansible playbook using it. Thus, a backwards compatible new version of the
-zhmc-ansible-modules package means that the user can safely upgrade to that new
+Ansible playbook using it. Thus, a backwards compatible new version of an
+Ansible Galaxy collection means that the user can safely upgrade to that new
 version without encountering compatibility issues in any Ansible playbooks
-using these modules.
+using the modules in the collection.
 
-This package uses the rules of `Semantic Versioning 2.0.0`_ for compatibility
+This collection uses the rules of `Semantic Versioning 2.0.0`_ for compatibility
```
Comment (sabyadi) [inline docs/source/intro.rst:361]: Broken link.

File: docs/source/intro.rst [hunk 7]
```diff
@@ -396,17 +353,17 @@ Compatibility
...
 
-The public interface of this package that is subject to the semantic versioning
-rules (and specificically to its compatibility rules) are the Ansible module
-interfaces described in this documentation.
+The public interface of the collection that is subject to the semantic
+versioning rules (and specificically to its compatibility rules) are the Ansible
+module interfaces described in this documentation.
 
 Violations of these compatibility rules are described in section
 :ref:`Change log`.
```
Comment (sabyadi) [inline docs/source/intro.rst:369]: Broken link.

**Review by sabyadi — COMMENTED** [review]
Please have a look at my comments below.

File: docs/source/intro.rst (hunk 1 above)
Comment (andy-maier) [reply docs/source/intro.rst:222]: DONE

File: docs/source/intro.rst (hunk 3 above)
Comment (andy-maier) [reply docs/source/intro.rst:337]: Yes. It is a standard placeholder resolved by Sphinx to the variable "release" in conf.py.

Comment (andy-maier) [reply docs/source/intro.rst:337]: I just saw that the version is no longer shown in the top left corner, and I have changed that statement here accordingly.

File: docs/source/intro.rst (hunk 4 above)
Comment (andy-maier) [reply docs/source/intro.rst:362]: For me, it works. It references an anchor with that name in the next line. This makes the link reusable for multiple places that want to have it. Although I must admit that this one is likely to be used only here.

File: docs/source/intro.rst (hunk 5 above)
Comment (andy-maier) [reply docs/source/intro.rst:339]: correct, I also just realized that. Removed the statement.

File: docs/source/intro.rst (hunk 7 above)
Comment (andy-maier) [reply docs/source/intro.rst:369]: Link is resolved by sphinx and works.

File: docs/source/intro.rst (hunk 2 above)
Comment (andy-maier) [reply docs/source/intro.rst:225]: no. will try it out.

Comment (andy-maier) [reply docs/source/intro.rst]: It does not. I modified this section to have the user invoke pip on the provided requirements file. however. the requirements file is not part of the collection archive, so we need to add that.

**Review by sabyadi — APPROVED** [review]

**Pull request merged**